src/System.Management.Automation/engine/GetEvent_Types_Ps1Xml.cs

```diff
@@ -1,90 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-macos`
Built-in type information tests.Should have correct number of built-in type items in type table
```powershell
Expected exactly 273, but got 271.
at <ScriptBlock>, /Users/runner/runners/2.159.2/work/1/s/test/powershell/engine/ETS/TypeTable.Tests.ps1: line 21
21:         $types.Count | Should -BeExactly 273
```

Reply:
There are 2 types only defined on Windows --
`PropertyCollection` and `ActiveDirectory.Entry`